<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoCode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="geocodeBtn">
     <property name="text">
      <string>Geocode</string>
     </property>
     <property name="shortcut">
      <string>F1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="reverseGeocodeBtn">
     <property name="text">
      <string>Reverse Geocode</string>
     </property>
     <property name="shortcut">
      <string>F2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QWebEngineView" name="map" native="true"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="upBtn">
     <property name="text">
      <string>Up</string>
     </property>
     <property name="shortcut">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="leftBtn">
     <property name="text">
      <string>Left</string>
     </property>
     <property name="shortcut">
      <string>Left</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="homeBtn">
     <property name="text">
      <string>Home</string>
     </property>
     <property name="shortcut">
      <string>Home</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="rightBtn">
     <property name="text">
      <string>Right</string>
     </property>
     <property name="shortcut">
      <string>Right</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="downBtn">
     <property name="text">
      <string>Down</string>
     </property>
     <property name="shortcut">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="zoomInBtn">
     <property name="text">
      <string>Zoom In</string>
     </property>
     <property name="shortcut">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="zoomOutBtn">
     <property name="text">
      <string>Zoom Out</string>
     </property>
     <property name="shortcut">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
